aws/resource_aws_efs_file_system.go

```diff
@@ -297,6 +338,17 @@ func resourceAwsEfsFileSystemRead(d *schema.ResourceData, meta interface{}) erro
 		return fmt.Errorf("error setting dns_name: %s", err)
 	}
 
+	res, err := conn.DescribeLifecycleConfiguration(&efs.DescribeLifecycleConfigurationInput{
+		FileSystemId: fs.FileSystemId,
+	})
+	if err != nil {
+		return fmt.Errorf("Error describing lifecycle configuration for EFS file system (%s): %s",
+			aws.StringValue(fs.FileSystemId), err.Error())
```

Comment:
Calling Error() on awserr objects is extraneous when printing error messages. 👍 

```suggestion
			aws.StringValue(fs.FileSystemId), err)
```